ted_sws/metadata_normaliser/services/xml_manifestation_metadata_extractor.py

```diff
@@ -263,7 +263,7 @@ def _get_normalised_namespaces(self):
         namespaces = dict([node for _, node in ET.iterparse(source=StringIO(self.xml_manifestation.object_data),
                                                             events=['start-ns'])])
 
-        namespaces["manifestation_ns"] = namespaces.pop("")
+        namespaces["manifestation_ns"] = namespaces.pop("") if "" in namespaces.keys() else ""
```

Comment:
Why do we remove the default NS? If that is for Semantic engineers, then we no longer need it. the XPath works with default namespace now.

Reply:
I had a notice that didn't have namespace from the test data in the F03 package. So it is a safety net in case it will come without namespace